cmd/adsysd/integration_tests/adsysctl_policy_test.go

```diff
@@ -594,6 +595,25 @@ func TestPolicyUpdate(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		// This is will make the scripts fail as well, sadly. But currently there's
```

Comment:
This is a little annoying. We could "fix" by restoring a run/users/uid directory with a mounts file and then make the file read only, rather than using localhost-uptodate or by changing the way we save the mounts file. 
Wdyt?

Reply:
Isn’t that the goal of `readOnlyDirs` that you are using? (Could be extended to files, but having the directory RO itself should still fail the mount manager AND scripts btw)

Reply:
Well, yes. But what I meant is that by making the run directory read only, we are making the scripts manager fail as well. Would be better if there was a way to make the mounts manager fail individually rather than always affecting both the mounts and the scripts managers.

Reply:
Could we try in one test to create the mounts file preventively and make it RO?

Reply:
I think we can change the way we go about readOnly dirs to make it work. We can try to create the specified path (if it doesn't exist) before making it read only. This would also give us more flexibility in the tests as well, I think.